### PR TITLE
bird1-openwrt: use a single Makefile for IPv4 and IPv6 packages

### PR DESCRIPTION
**Pull request by rogerpueyo**
Having two Makefile files, respectively in the bird1-openwrt-ipv4 and
bird1-ipv6-openwrt folders, made the commit tests fail due to a
compilation error at the time of building the packages. This patch
merges both files into a single Makefile. As a result, package
compilation succeeds and all commit tests can be passed.

Signed-off-by: Roger Pueyo Centelles <roger.pueyo@guifi.net>

### CODE REVIEW

**Comment by mwarning** [comment]
There seems to be some problem with the ucode package on powerpc_8540

**Comment by rogerpueyo** [comment]
Hi, @mwarning,

> There seems to be some problem with the ucode package on powerpc_8540

I believe the problem is caused by the `ucode` package itself, and only affects the powerpc_8540 architecture. Indeed, even an [empty commit](https://github.com/rogerpueyo/openwrt-routing-packages/actions/runs/4072096853/jobs/7014516919) triggers the same error, and only on that same architecture.

I'd kindly ask you --or any other person with permissions to do it-- to consider merging the commit, even if the `powerpc_8540` test fails, as the error is unrelated to the `bird1-openwrt` package. This way, I'll be able to update PRs #921 and #923 accordingly.

For your convenience, I just rebased with upstream/master.

Thanks!

**Comment by mwarning** [comment]
hi, let's wait for the current pipeline to run. Let's merge then.

File: bird1-openwrt/Makefile [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  TITLE+= (IPv6)
+  DEPENDS+= +$(PKG_IPV6_UCI_NAME)
+endef
+
+define Package/$(PKG_IPV4_UCI_NAME)/conffiles
+	/etc/config/$(BIRD4)
+endef
+
+define Package/$(PKG_IPV6_UCI_NAME)/conffiles
+	/etc/config/$(BIRD6)
```
Comment (BKPepe) [inline bird1-openwrt/Makefile:106]: I am afraid that this is wrong. The Conffiles section does not work with the tab. There should be no indentation at all.

File: bird1-openwrt/Makefile [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+	$(INSTALL_DIR) $(1)/etc/$(BIRD6)/filters
+	$(INSTALL_DIR) $(1)/etc/$(BIRD6)/functions
+	$(INSTALL_BIN) ./$(PKG_IPV6_NAME)/src/init.d/$(BIRD6)* $(1)/etc/$(BIRD6)/init.d/
+	$(CP) ./$(PKG_IPV6_NAME)/src/uci-defaults/* $(1)/etc/$(BIRD6)/init.d/
+	$(INSTALL_CONF) ./$(PKG_IPV6_NAME)/src/config/$(BIRD6) $(1)/etc/config/
+endef
+
+define Package/$(PKG_IPV4_UCI_NAME)/postinst
+#!/bin/sh
+    if [ -z "$${IPKG_INSTROOT}" ]; then
```
Comment (BKPepe) [inline bird1-openwrt/Makefile:131]: Same, also no indentation.

File: bird1-openwrt/Makefile [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+endef
+
+define Package/$(PKG_IPV6_LUCI_NAME)/install
+	$(INSTALL_DIR) $(1)/usr/lib/lua/luci/controller/
+	$(INSTALL_DIR) $(1)/usr/lib/lua/luci/model/cbi/$(BIRD6)/
+	$(INSTALL_DIR) $(1)/usr/lib/lua/luci/view/$(BIRD6)/
+	$(CP) ./$(PKG_IPV6_NAME)/src/model/* $(1)/usr/lib/lua/luci/model/cbi/$(BIRD6)/
+	$(CP) ./$(PKG_IPV6_NAME)/src/controller/* $(1)/usr/lib/lua/luci/controller/
+	$(CP) ./$(PKG_IPV6_NAME)/src/view/* $(1)/usr/lib/lua/luci/view/$(BIRD6)/
+endef
```
Comment (BKPepe) [inline bird1-openwrt/Makefile:174]: The ``luci-app-bird`` should be moved to LuCI repository.

File: bird1-openwrt/Makefile [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+BIRD4 := bird4
+BIRD6 := bird6
+
+PKG_NAME := $(BIRD1_PKG)-openwrt
+PKG_IPV4_NAME := $(BIRD1_IPV4_PKG)-openwrt
+PKG_IPV6_NAME := $(BIRD1_IPV6_PKG)-openwrt
+PKG_VERSION := 0.3
+PKG_RELEASE := 2
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+PKG_LICENSE := GPL-3.0+
```
Comment (BKPepe) [inline bird1-openwrt/Makefile:32]: This is a deprecated SPDX license identifier.

**Pull request merged**